src/containers/shared/components/TxDetails.tsx

```diff
@@ -61,116 +59,22 @@ interface Props {
 }
 
 const TxDetails = (props: Props) => {
-  const language = useLanguage()
   const { t } = useTranslation()
 
   function renderAmount(d: any): ReactElement {
     return <Amount value={d} />
   }
 
-  function renderEscrowFinish(): ReactElement {
+  function renderSignerListSet(): ReactElement {
```

Comment:
@ckniffen shouldn't the SignerListSet be removed from here too now?

Reply:
This seems like a merge/rebase error

Reply:
yeah I'm looking into it